message.py

```diff
@@ -8,18 +9,18 @@ def send_email(sender, password, receiver, message):
         server.login(sender, password)
         server.sendmail(sender, receiver, message)
 
-        return "The message was sent succesfuly!"
+        return "The message was sent successfully!"
     except Exception as _ex:
         return f"{_ex}\nCheck your login or password please!"
 
-
 def main():
-    sender = "boyhich3@gmail.com"
-    password = "tmtzabdqcjmmrpwu"
-    receiver = "shtrikker28@gmail.com"
-    message = input("Type your message: ")
-    print(send_email(sender, password, receiver, message))
-
+    parser = argparse.ArgumentParser(description='Sending email')
+    parser.add_argument('-s', dest="sender", help='sender')
+    parser.add_argument('-p', dest="password", help='password')
+    parser.add_argument('-r', dest="receiver", help='receiver')
+    parser.add_argument('-m', dest="message", help='message')
+    args = parser.parse_args()
+    send_email(sender=args.sender, password=args.password, receiver=args.receiver, message=args.message)
```

Comment:
Too long row, you can remove sender=*, ...
Better rewrite to _send_email(args.sender, args.password, args.receiver, args.message)_